ds_judgements_public_ui/sass/includes/_service_introduction.scss

```diff
@@ -1,16 +1,22 @@
 .service-introduction {
   background-color: $color-almost-black;
```

Comment:
I've noticed this is different to the main background colour. It's hard to see but if you look closely it's not the same:

<img width="525" alt="image" src="https://github.com/nationalarchives/ds-caselaw-public-ui/assets/173052089/77fe4791-4aa5-4fda-8299-55fc85465fa2">